controllers/storagecluster/generate.go

```diff
@@ -83,3 +83,11 @@ func generateCephReplicatedSpec(initData *ocsv1.StorageCluster, poolType string)
 func generateStorageQuotaName(storageClassName, quotaName string) string {
 	return fmt.Sprintf("ocs-%s-%s", storageClassName, quotaName)
 }
+
+func generateNameForDefaultStorageClassClaimFile(initData *ocsv1.StorageCluster) string {
+	return fmt.Sprintf("%s-claim-file", initData.Name)
```

Comment:
Let the name start with `storageclassclaim-%s-file/block`?

Reply:
done